src/PVRIptvData.cpp

```diff
@@ -401,6 +404,11 @@ bool PVRIptvData::LoadPlayList(void)
           strTvgLogo = strChnlName;
         }
 
+        if (!strChnlNo.empty()) 
+	{
```

Comment:
Thank you for answer. I do not know because I'm a beginner, do I have to modify the source and commit again?